dbt/tests/adapter/concurrency/test_concurrency.py

```diff
@@ -303,8 +303,6 @@ def models(self):
             "skip.sql": models__skip_sql,
         }
 
-
-class TestConcurenncy(BaseConcurrency):
```

Comment:
Did you cut this one off.

Reply:
Yeah, I effectively moved the test method from the test class up into the base class.